docs/coins.md

```diff
@@ -48,6 +48,7 @@ This list is generated from [./coins.json](../coins.json)
 | 888     | NEO              | NEO    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/neo/info/logo.png" width="32" />          | <https://neo.org>             |
 | 889     | TomoChain        | TOMO   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/tomochain/info/logo.png" width="32" />    | <https://tomochain.com>       |
 | 1001    | Thunder Token    | TT     | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/thundertoken/info/logo.png" width="32" /> | <https://thundercore.com>     |
+| 1023    | Harmony          | ONE    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/harmony/info/logo.png" width="32" />      | <https://harmony.one>         |
```

Comment:
https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/harmony/info/logo.png the image is not there.

Reply:
yes, aware: I didn't get to this repo's respective coming PR https://github.com/trustwallet/assets#repository-structure

Reply:
can you upload your logo to  https://github.com/trustwallet/assets?